ts/src/binanceus.ts

```diff
@@ -102,6 +103,28 @@ export default class binanceus extends binance {
                         'status': 1,
                     },
                 },
+                'sapi': {
+                    'get': {
+                        'apiTradingStatus': 1,
+                        'asset/assetDistributionHistory': 1,
+                        'asset/query/trading-fee': 1,
+                        'asset/query/trading-volume': 1,
+                        'sub-account​/spotSummary': 1,
+                        'sub-account​/status': 1,
+                    },
+                },
+                'sapiV3': {
```

Comment:
hum actually we have already a sapiV3 in binance 
![image](https://github.com/ccxt/ccxt/assets/43336371/35bc3312-a7cc-4e29-aa9e-38b1b81757e3)